contrib/kind.sh

```diff
@@ -80,6 +80,7 @@ usage() {
     echo "                 [-is | --ipsec]"
     echo "                 [-cm | --compact-mode]"
     echo "                 [-ic | --enable-interconnect]"
+    echo "                 [-cnc | --custom-net-conf-enable]"
```

Comment:
maybe we should make the name of the knob something related to the OKEP name ?

Reply:
> The actual feature gate name is enable-custom-network-configuration

Oh, this is the name of the OpenShift feature gate ? Ok, that kind of changes the picture :) 

I guess I'm OK with it then. Maybe be a tad more verbose - `custom-net-config-enable` ?

Reply:
This is for OVN-K, the one is OCP is [PreconfiguredUDNAddresses](https://github.com/openshift/api/pull/2348)

Reply:
Resolving this, this naming suggestion is OK.